base_classes/NXparameters.nxdl.xml

```diff
@@ -26,11 +26,69 @@
     xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
     name="NXparameters" 
 	type="group" extends="NXobject">
-    <doc>Container for parameters, usually used in processing or analysis.</doc>
+    <doc>
+        Container for parameters used in processing or analysing data.
+
+        This base class can be used in a number of different contexts,
+        and is allowed in any other base class. However, a common use
+        case is to store the results of optimizing parameters, for
+        example, by least-squares fitting, so a number of attributes
+        have been defined for this purpose. Normally, this group will be
+        stored in a :ref:NXprocess group, which defines the program used
+        to process these parameters. If multiple functions are used to
+        analyze the data, there can be additional NXparameter groups for
+        each one.        
+    </doc>
+    <attribute name="model">
+        <doc>
+            The name of the model used in optimizing the data. Fitting
+            packages such as LMFIT (https://lmfit.github.io/lmfit-py/)
+            provide models, which instantiate functions to be fitted to
+            the data. If this attribute is provided, it is assumed that
+            all the parameters in this group are associated with this
+            model.
+        </doc>
+    </attribute>
     <field name="TERM" minOccurs="0" maxOccurs="unbounded" nameType="any">
-        <!-- maxOccurs="unbounded" is intended but is not allowed by current syntax -->
-        <doc>A parameter (also known as a term) that is used in or results from processing.</doc>
-        <attribute name="units"/>
+        <doc>
+            A parameter (also known as a term) that is used in or
+            results from processing. 
+        </doc>
+        <attribute name="units" />
+        <attribute name="error" type="NX_NUMBER">
```

Comment:
We have the reserved suffix "_errors" why not use this?
Like parameter called "alpha" and given that errors exists implicitly we also have "alpha_errors"
https://manual.nexusformat.org/datarules.html